public/docs/_examples/intro/dart/lib/hero_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.hero_service;
+
+import 'package:angular2/angular2.dart';
+import 'package:developer_guide_intro/hero.dart';
+import 'package:developer_guide_intro/backend_service.dart';
+import 'package:developer_guide_intro/logger_service.dart';
+
+@Injectable()
+class HeroService {
+  var _backendService;
+  var _logger;
+  HeroService(){
+    _logger = new Logger();
+    _backendService = new BackendService();
+  }
+  getHeroes(){
+    List<Hero> heroes = _backendService.getAll(Hero);
```

Comment:
It does run, but I agree with you. The TS example had the extra `let heroes`. Sets it then returns it. I didn't see a reason for it but converted it over 'as is' just in case I was missing something. I'll update it.